src/downloadmanager.h

```diff
@@ -405,6 +405,7 @@ class DownloadManager : public QObject
 public:  // IDownloadManager interface:
   int startDownloadURLs(const QStringList& urls);
   int startDownloadNexusFile(int modID, int fileID);
```

Comment:
If this function is not used in the main code, this should be removed an the proxy should directly call `startDownloadNexusFileForGame` from `startDownloadNexusFile` (cannot check right now if that's the case or not).